### ISSUE TITLE
Support RFC 4437 (Redirects)

### ISSUE THREAD

**Issue by jwodder** [dandi/dandidav#28]
[RFC 4437](http://www.webdav.org/specs/rfc4437.html) defines the `<DAV:reftarget>` property, which can be used to inform clients where non-collection resources redirect to.

**Comment by yarikoptic** [comment]
That sounds like might be valuable since most of our targets are redirects, right?

**Comment by jwodder** [comment]
@yarikoptic Yes, though I believe the best effect from adding this would be that clients that support RFC 4437 would download redirected files by going directly to the redirect URL rather than making a request to `dandidav` that then has to be processed into a redirect.
